suzieq/poller/controller/source/netbox.py

```diff
@@ -115,10 +115,16 @@ def _init_session(self, headers: dict):
             headers ([dict]): headers to initialize the session
         """
         if not self._session:
-            self._session = aiohttp.ClientSession(
-                headers=headers,
-                connector=aiohttp.TCPConnector(verify_ssl=self._ssl_verify)
-            )
+            if self._ssl_verify:
+                self._session = aiohttp.ClientSession(
+                    headers=headers,
+                    connector=aiohttp.TCPConnector(ssl=None)
+                )
+            else:
+                self._session = aiohttp.ClientSession(
+                    headers=headers,
+                    connector=aiohttp.TCPConnector(ssl=False)
+                )
```

Comment:
Why not simply set ssl to self._ssl_verify?

Reply:
I set the ssl=False instead of _ssl_verify because it was easier to read. The only way the code will enter in this code is with self._ssl_verify = False